### DISCUSSION TITLE
`ntpd-rs` now the default in Serpent OS

### DISCUSSION THREAD

**Discussion by ikeycode** [AerynOS/discussions#11] [Announcements]
Hot on the trail of yesterdays coreutils (uutils) swap, after a `sudo moss sync -u`, you'll see our `systemd` package pulling in `ntpd-rs` as a runtime dependency.

This, through presets, will make `ntpd-rs` (ntp-daemon) the default NTP handler in Serpent OS, rather than `timesyncd`.

![Screenshot From 2024-09-23 22-34-57](https://github.com/user-attachments/assets/716d07d4-d4e7-48b5-9b11-c620f36fd890)

**Comment by tristan957** [comment]
Are there any benefits over timesyncd?

**Reply by ikeycode** [reply]
For some context, see https://letsencrypt.org/2024/06/24/ntpd-rs-deployment.html
